service.libraryautoupdate/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyo
```

Comment:
Please remove .gitignore file

Reply:
Will do, I always forget that thing when merging in the changes to the repo-scripts repository.

Reply:
i don't have problems with gitignore files as they don't hurt (they might even help)